### DISCUSSION TITLE
Message.Id and Message.CorrelationId are declared as Guid

### DISCUSSION THREAD

**Discussion by IlSocio** [BrighterCommand/Brighter#2067] [Q&A] [answered]
Hello everyone... 
I just noticed that Brighter is forcing these two fields to be Guid, while when we use directly the transports ASB or Rabbit, it doesn't exists this limitation.

Would you accept a PR to use the more generic "string" type instead of Guid?

In case that the string is not provided, it could still fallback to the default value of Guid.NewGuid().ToString(), but, at least, we have the opportunity to provide any string value for the MessageId and CorrelationId, without any particular restriction.

**Comment by iancooper** [comment, marked as answer]
HI @IlSocio,

Sorry I missed this, don't seem to be getting discussion notifications. It's not a bad point, we could think about it. We would need to decide if it was breaking. If it was it would have to go into our roadmap for v10. But I agree that we should probably be more flexible here

**Comment by iancooper** [comment]
Going to create a v10 issue from the discussion
